src/app/src/user/services/UserRepository.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * This file is part of the GZERO CMS package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Class UserController
+ *
+ * @package    Admin
+ * @author     Mateusz Urbanowicz <urbanowiczmateusz89@gmail.com>
+ * @copyright  Copyright (c) 2015, Mateusz Urbanowicz
+ */
+
+'use strict';
+
+function UserRepository(Restangular) {
+    var api = 'admin/users';
+    var contents = Restangular.all(api);
```

Comment:
Name of the variable should correspond to api resource, so i think that the name should be 'users' instead of 'contents'.